src/Bicep.LangServer.IntegrationTests/Registry/ModuleRestoreSchedulerTests.cs

```diff
@@ -97,9 +97,9 @@ public async Task RestoreShouldBeScheduledAsRequested()
             var thirdSource = new TaskCompletionSource<bool>();
 
             var compilationManager = Repository.Create<ICompilationManager>();
-            compilationManager.Setup(m => m.RefreshCompilation(firstUri)).Callback<DocumentUri>(uri => firstSource.SetResult(true));
-            compilationManager.Setup(m => m.RefreshCompilation(secondUri)).Callback<DocumentUri>(uri => secondSource.SetResult(true));
-            compilationManager.Setup(m => m.RefreshCompilation(thirdUri)).Callback<DocumentUri>(uri => thirdSource.SetResult(true));
+            compilationManager.Setup(m => m.RefreshCompilation(firstUri, false)).Callback<DocumentUri, bool>((uri, _) => firstSource.SetResult(true));
```

Comment:
I presume the choice to not default to false in the function args was deliberate. As a learning opportunity for me, can you share your considerations ? I would likely have left a default value since forcing refresh is an opt-in operation in my mind.

Reply:
I'm personally very opinionated about default arguments - they can be a source of ambiguity and bugs, because different people may have different expectations about what the "default behavior" is (especially through period of refactoring where expectations can change).

Unless it's extremely unambiguous, I generally err on the side of explicitness to make the behavior clearer to other people reading the code.

Reply:
perhaps then qualifying the argument name would be helpful. 
```c#
 compilationManager.Setup(m => 
    m.RefreshCompilation(firstUri, forceReloadAuxiliaryFiles: false))
    .Callback<DocumentUri, bool>((uri, _) => firstSource.SetResult(true));
```